### ISSUE TITLE
Omitted implied 1x source untested

### ISSUE THREAD

**Issue by Wilto** [yoavweiss/Blink#9]
Just having a look through https://github.com/yoavweiss/Blink/blob/master/Source/core/html/parser/HTMLSrcsetParserTest.cpp and it doesn’t look like there’s a test for an omitted `1x`flag:

`1x.gif, 2x.gif 2x`

**Comment by yoavweiss** [comment]
You're right. I'll add one (It's tested in the layout tests, but that's no excuse)

**Comment by Wilto** [comment]
I probably could’ve attempted to add it myself, but you know what would happen to the entire rendering engine if I committed code anywhere near it.
